pvlib/pvsystem.py

```diff
@@ -1840,44 +1853,69 @@ def sapm_spectral_loss(airmass_absolute, module):
 
 def sapm_effective_irradiance(poa_direct, poa_diffuse, airmass_absolute, aoi,
                               module, reference_irradiance=1000):
-    """
+    r"""
     Calculates the SAPM effective irradiance using the SAPM spectral
     loss and SAPM angle of incidence loss functions.
 
     Parameters
     ----------
     poa_direct : numeric
-        The direct irradiance incident upon the module.
+        The direct irradiance incident upon the module. [W/m2]
 
     poa_diffuse : numeric
-        The diffuse irradiance incident on module.
+        The diffuse irradiance incident on module.  [W/m2]
 
     airmass_absolute : numeric
-        Absolute airmass.
+        Absolute airmass. [unitless]
 
     aoi : numeric
-        Angle of incidence in degrees.
+        Angle of incidence. [degrees]
 
     module : dict-like
         A dict, Series, or DataFrame defining the SAPM performance
         parameters. See the :py:func:`sapm` notes section for more
         details.
 
-    reference_irradiance : numeric, default 1000
```

Comment:
removed doc string but did not remove from function signature.

Reply:
what's the logic behind removing the kwarg from this function and adding it to the other? I suggest we should consistently provide flexibility or consistently enforce W/m^2.

Reply:
Removed from signature (was my oversight) to be consistent with the input expected by other irradiance-to-DC power functions, i.e., effective irradiance in W/m2. SAPM was the oddball.